plugins/chunter-resources/src/index.ts

```diff
@@ -80,6 +81,58 @@ export async function UnpinMessage (message: ChunterMessage): Promise<void> {
   })
 }
 
+export async function ArchiveChannel (channel: Channel, afterArchive?: ()=>void): Promise<void> {
+  showPopup(
+    MessageBox,
+    {
+      label: chunter.string.ArchiveChannel,
+      message: chunter.string.ArchiveConfirm
+    },
+    undefined,
+    (result: boolean) => {
+      if (result) {
+        const client = getClient()
+
+        // eslint-disable-next-line @typescript-eslint/no-floating-promises
+        client.update(channel, { archived: true })
+        if (afterArchive)
+          afterArchive()
+
+        const loc = getCurrentLocation()
+
+        if (loc.path.includes(channel._id) ) {
+          const path = loc.path.slice(0, 2)
```

Comment:
Why you change path so strange? You can just set `loc.path.length = 2`

Reply:
Fixed.
Added issue https://github.com/hcengineering/anticrm/issues/1431